inference-engine/tests/functional/plugin/cpu/shared_tests_instances/single_layer_tests/extract_image_patches.cpp

```diff
@@ -0,0 +1,42 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <vector>
+
+#include "single_layer_tests/extract_image_patches.hpp"
+
+using namespace LayerTestsDefinitions;
+using ngraph::op::PadType;
+
+namespace {
+
+const std::vector<std::vector<size_t>> inDataShape = {{1, 1, 10, 10}, {1, 3, 10, 10}};
+const std::vector<std::vector<size_t>> kernels = {{2, 2}, {3, 3}, {4, 4}};
+const std::vector<std::vector<size_t>> strides = {{3, 3}, {5, 5}, {8, 8}, {9, 9}};
+const std::vector<std::vector<size_t>> rates = {{1, 1}, {2, 2}};
```

Comment:
Please also add assymetric between Height and Width cases. This is necessary to check we don't have mistake in indexing inside the impl.

Reply:
Done